CHANGELOG.md

```diff
@@ -1,5 +1,176 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
+NOTE 1: Requires MacOS v10.14 (Mojave) to build.
```

Comment:
```suggestion
NOTE: This version does not support Realm Object Server or cloud.realm.io. Use v6.0 if you need that.
NOTE: This version bumps the Realm file format to version 11. Files created with previous versions of Realm will be automatically upgraded. It is not possible to downgrade to earlier versions.
```

Reply:
We have a pending update coming in from `v10-latest`.